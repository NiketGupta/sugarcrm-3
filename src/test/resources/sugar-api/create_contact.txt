method=set_entry
&input_type=json
&response_type=json
&rest_data={
   "session":"blq8sg90fb11sv3fbfkjsk1am4",
   "module_name":"Contacts",
   "name_value_list":[
      {
         "name":"first_name",
         "value":"John"
      },
      {
         "name":"last_name",
         "last_name":"Braithwaite"
      }
   ]
}
