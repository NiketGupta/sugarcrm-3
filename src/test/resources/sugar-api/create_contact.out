{"id":"9eb94da9-e61a-915b-30e1-52b341a17fe0","entry_list":{"first_name":{"name":"first_name","value":"John"}}}